gemdaq-testing/gemsupervisor/include/gem/supervisor/tbutils/ADCScan.h

```diff
@@ -56,8 +56,6 @@ namespace gem {
     }
   }
   
-  typedef std::shared_ptr<hw::vfat::HwVFAT2 > vfat_shared_ptr;
```

Comment:
Don't know why this got removed from yours, double check that it is in your release-v1 branch